### PR TITLE
[KO Number] Integer and NumberRange Coverage Improvement

### PR DESCRIPTION
**Pull request by neudurgeshp**
**PR Includes**
- Number Definitions

**Number Extractor (Integer Improvement)**
- Total Cases 244: (Extractor Passed - 226, Full Passed - 101, Skipped - 18)
- New Test Spec: Cases 135 (Extractor Passed - 128, Full Passed - 4, Skipped - 7)
- Integer 56: Extraction Passed-55, Skipped-1
- Fraction 58: Extraction Passed-58
- Decimal 9: Extraction Passed-9
- Power 2: Extraction Passed-2
- Negative 10 : Extraction Passed-4, Skipped-6

**Number Range Extractor**

- Total Test Cases 122: (Extractor Passed - 85, Full Passed - 64, Skipped - 37)
- Existing Cases 53: Extractor Passed - 26, Full Passed - 22, Skipped- 27
- New Cases 69: Extractor Passed - 59, Full Passed - 42, Skipped - 10

**Case input modified **"이 곱하기 십삼은 이십육이다"** to **"내가 가장 좋아하는 숫자는 이십육이다"**

**Reason:** Earlier input had 3 numbers but there was only 1 text in result set. So need to be modified either input or result set.

### CODE REVIEW

**Comment by tellarin** [comment]
@neudurgeshp, can you solve the merge conflict? Thanks.

**Comment by neudurgeshp** [comment]
> @neudurgeshp, can you solve the merge conflict? Thanks.
Sure. I'll resolve.

**Comment by neudurgeshp** [comment]
> @neudurgeshp, can you solve the merge conflict? Thanks.

Resolved.

File: .NET/Microsoft.Recognizers.Definitions.Common/Korean/NumbersDefinitions.cs [hunk 1]
```diff
@@ -1,243 +1,247 @@
...
-            { @"마지막의 옆", @"end" },
-            { @"마지막에서 바로 전의 것", @"end" },
-            { @"이전 것", @"current" },
-            { @"다음 것", @"current" },
-            { @"마지막에서 바로 전", @"end" },
-            { @"지금의 것", @"current" },
-            { @"현재", @"current" }
-        };
-    }
+//------------------------------------------------------------------------------
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Definitions.Common/Korean/NumbersDefinitions.cs:1]: Please be careful to not change the line endings of a file unless necessary. Can you revert this change and only push the conflict resolution?

Comment (neudurgeshp) [reply .NET/Microsoft.Recognizers.Definitions.Common/Korean/NumbersDefinitions.cs:1]: Sure, it was auto-formatted. I'll take care for future PR/Conflicts.

Comment (tellarin) [reply .NET/Microsoft.Recognizers.Definitions.Common/Korean/NumbersDefinitions.cs:1]: I'm still seeing the line end changes after the last update.

Comment (neudurgeshp) [reply .NET/Microsoft.Recognizers.Definitions.Common/Korean/NumbersDefinitions.cs:1]: Where is that exactly? My be I'm referring something else.

Comment (tellarin) [reply .NET/Microsoft.Recognizers.Definitions.Common/Korean/NumbersDefinitions.cs:1]: The whole NumbersDefinitions.cs file included in this PR after the merge conflict was resolved shows as having the line endings changed when compared to the one on /master.
On the web UI you can see it in "Files changed". NumbersDefinitons.cs shows as having  1,243 lines deleted and re-added.

**Comment by neudurgeshp** [comment]
Closing this pull request because of unstable branch. Raising new PR
